Jenkinsfile.baguette

```diff
@@ -84,11 +97,16 @@ pipeline {
                     agent {
                         label 'team-local && linux'
                     }
-		    environment {
-			DOCKERAPP_BINARY = '../docker-app-linux'
-		    }
+            environment {
+                DOCKERAPP_BINARY = '../docker-app-linux'
+            }
                     steps  {
+                        dir('/tmp') {
+                            unstash "invocation-image"
```

Comment:
Please can we remove the use of `/tmp` from `Jenkinsfile.baguette` such that everything is within the workspace (`Jenkins` has it right I think).

In doing so I believe you will be able to avoid the need to version the file and therefore avoid needing `make` on the workers node to call `load-invocation-image` (which is just there to resolve the variables in a single place) since you can just do a `docker load` in the script fragment below instead.

Reply:
Yep totally I'm fixing it 👍

Reply:
Fixed, PTAL @ijc